stubs/humanfriendly/humanfriendly/case.pyi

```diff
@@ -1,9 +1,9 @@
-import collections
+from collections import OrderedDict
 from typing import Any
 
 from humanfriendly.compat import unicode
 
-class CaseInsensitiveDict(collections.OrderedDict):
+class CaseInsensitiveDict(OrderedDict[Any, Any]):
```

Comment:
Maybe `CaseInsensitiveDict` should be a generic class? It seems to treat non-string keys just like `OrderedDict`.